apps/fabric-website/src/SiteDefinition/SiteDefinition.pages/Controls/ios.tsx

```diff
@@ -116,6 +125,30 @@ export const controlsPagesIos: INavPage[] = [
       },
     ],
   },
+  {
+    title: 'Notification & Engagement',
+    isCategory: true,
+    pages: [
+      {
+        title: 'MessageBar',
```

Comment:
To match the style used for other iOS controls:
```suggestion
        title: 'Message Bar',
```